source/content.ts

```diff
@@ -107,6 +107,7 @@ import './features/highest-rated-comment';
 import './features/clean-issue-filters';
 import './features/minimize-upload-bar';
 import './features/cycle-lists-with-keyboard-shortcuts';
+import './features/highlight-file-in-prs';
```

Comment:
Maybe:

- `open-prs-for-file` (but it sounds like a verb *to open* a PR)
- `list-prs-for-file`
- `list-prs-with-file`
- `list-prs-editing-current-file`

Reply:
I used the word `highlight` because it's design is somewhat related to [`highlight-closing-prs-in-open-issues`](https://github.com/sindresorhus/refined-github/blob/master/source/features/highlight-closing-prs-in-open-issues.tsx).

What about:
* `highlight-file-altered-in-pr`
* `file-altered-in-pr`
* `altered-in-pr`
* `file-used-in-pr`

Reply:
In that case `highlight` kinda works because it brings the same information from the bottom of the thread to the top. In this case this information is fetched from elsewhere. 

This feature is about showing a list of PRs related to a file, so wouldn’t it make sense to focus the name on that? Technically it’s `show-prs-containing-the-current-file`, not the other way around. We’re not “highlighting files”, we’re “adding links to PRs”

Reply:
What about `highlight-affected-prs-in-file`...

The more I think about this one, the more it has my preference.

Reply:
I went with `highlight-affected-prs-in-file`. The name is long, but it captures exactly what it does.